src/libs/actions/CompanyCards.ts

```diff
@@ -740,6 +725,29 @@ function openPolicyCompanyCardsFeed(policyID: string, feed: CompanyCardFeed) {
     API.read(READ_COMMANDS.OPEN_POLICY_COMPANY_CARDS_FEED, parameters);
 }
 
+function flatAllCardsList(allCardsList: OnyxCollection<WorkspaceCardsList>, workspaceAccountID: number): Record<string, Card> | undefined {
```

Comment:
I think this method would deserve some plain English explanation of what its used for and what is the expected output format (unit test possibly too)